sklearn/ensemble/stacking.py

```diff
@@ -0,0 +1,265 @@
+"""Stacked ensembles"""
+
+# Author: Caio Oliveira <caioaao@gmail.com>
+# License: BSD 3 clause
+
+from ..base import (BaseEstimator, TransformerMixin, MetaEstimatorMixin, clone)
+from ..model_selection import cross_val_predict
+from ..pipeline import FeatureUnion
+from ..preprocessing import FunctionTransformer
+
+
+class StackingTransformer(BaseEstimator, MetaEstimatorMixin, TransformerMixin):
+    """Transformer to turn estimators into meta-estimators for model stacking
+
+    In stacked generalization, meta estimators are combined in layers to
```

Comment:
Uhm I am confused.
I read in several place that the meta-learner is the one combining the base estimator. So it is a bit weird to see meta estimators here.